implementation/contracts/deposit/DepositLiquidation.sol

```diff
@@ -95,7 +95,7 @@ library DepositLiquidation {
 
         _d.liquidationInitiated = block.timestamp;  // Store the timestamp for auction
         _d.liquidationInitiator = msg.sender;
-        _d.setFraudLiquidationInProgress();
+        _d.setLiquidationInProgress();
```

Comment:
Doesn't this change require an update in unit tests? Can we make sure we verify state in tests after calling this function (for `notifyUndercollateralizedLiquidation` and `notifyCourtesyTimeout`)?

Reply:
> Doesn't this change require an update in unit tests?

nope, we didn't verify state before.

> Can we make sure we verify state in tests after calling this function

Yes! adding state verification for `notifyUndercollateralizedLiquidation` `notifyCourtesyTimeout` `provideECDSAFraudProof` and `provideSPVFraudProof`